package.json

```diff
@@ -52,7 +55,8 @@
     "eslint-plugin-ember": "^5.2.0",
     "eslint-plugin-node": "^7.0.1",
     "loader.js": "^4.7.0",
-    "qunit-dom": "^0.8.0"
+    "qunit-dom": "^0.8.0",
+    "sinon": "^7.5.0"
```

Comment:
Where is it used?

Reply:
It's not, I have not included unit tests as they require much more work. Still not fully figured out how to approach them.

Reply:
Removed.